src/coreclr/jit/importercalls.cpp

```diff
@@ -3125,15 +3120,7 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
     // To be fixed in https://github.com/dotnet/runtime/pull/77465
     const bool tier0opts = !opts.compDbgCode && !opts.jitFlags->IsSet(JitFlags::JIT_FLAG_MIN_OPT);
 
```

Comment:
```suggestion
    const bool tier0opts = !opts.compDbgCode && !opts.jitFlags->IsSet(JitFlags::JIT_FLAG_MIN_OPT);

```
Unused? (Also, delete the comment too.)

Reply:
No, actually it should have been placed back in the `if (!mustExpand)` statement. I've put it back to how it was prior to #101800 when the explicitly `betterToExpand` handling was added